test/e2e/testdata/artifactgc/artgc-multi-strategy-multi-anno.yaml

```diff
@@ -1,9 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Workflow
 metadata:
-  generateName: two-artgc-
-  #finalizers:
-  #  - "blah"
+  generateName: testing-on-completion-and-on-deletion-strategies-
```

Comment:
made this workflow name really long as a way of testing this as part of the e2e test. I confirmed it fails on master and passes here.